wordpress-mobile/WordPress-iOS
# Zendesk Mobile: create identity when it's actually needed.

Fixes #9220 

To test:
- Fresh Install of the app.
- Log into a Self hosted site.
  - Open Me > Support.
  - Verify the three Support options work.
  - Find the `Zendesk identity created` log and verify the email and name match the site.
- Log into a WP account.
  - Open Me > Support.
  - Select one of the Support options.
  - Find the `Zendesk identity created` log and verify the email and name match the WP site.
- Log out of the WP account.
  - Open Me > Support.
  - Select one of the Support options.
  - Find the `Zendesk identity created` log and verify the email and name match the self hosted site.

To note: In the absence of a WP account, the app will use the _first_ self hosted site for Zendesk, regardless of additional self hosted sites added. Since this is written to User Defaults, a re-install is needed to clear it.
